config/maintainers.json

```diff
@@ -100,6 +100,16 @@
       "link_url": "https://www.typedspace.com/",
       "avatar_url": null,
       "bio": "Loosely typed person digging further into the stack."
-    }
+    },
+    {
+      "github_username": "workingjubilee",
+      "alumnus": false,
+      "show_on_website": false,
+      "name": "Jubilee Young",
+      "link_text": null,
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "Builder of rainbow bridges."
+    },
```

Comment:
```suggestion
    }
```

Reply:
I was saying to myself "surely this won't break a JSON parse?" and then I actually tried it.